src/test/java/uk/gov/ons/ctp/response/collection/exercise/endpoint/CollectionExerciseEndpointUnitTests.java

```diff
@@ -303,6 +306,49 @@ public void linkSampleUnitsPut() throws Exception {
         .andExpect(jsonPath("$.sampleSummaryIds[*]", containsInAnyOrder(SAMPLE_SUMMARY_ID1.toString(), SAMPLE_SUMMARY_ID2.toString())));
   }
 
+  /**
+   * Tests put request to unlink sample.
+   *
+   * @throws Exception exception thrown
+   */
+  @Test
+  public void testUnlinkSampleUnitsPut() throws Exception {
```

Comment:
Maybe remove the Put from the method

Reply:
I agree that removing the 'Put' is better naming style.